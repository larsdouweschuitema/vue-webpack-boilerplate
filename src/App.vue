<template>
  <div id="app">
    <h1>App</h1>
  </div>
</template>

<script>
  export default {
    name: "App"
  };
</script>