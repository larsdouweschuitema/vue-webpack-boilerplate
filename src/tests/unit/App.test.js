import { mount } from "@vue/test-utils";
import App from "@/App";

describe("App component", () => {
  test("is a Vue instance", () => {
    const wrapper = mount(App);
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
  const wrapper = mount(App)
  expect(wrapper.element).toMatchSnapshot()
  });
});
