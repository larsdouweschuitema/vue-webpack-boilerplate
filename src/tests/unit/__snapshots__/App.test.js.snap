// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div
  id="app"
>
  <h1>
    App
  </h1>
</div>
`;
